packages/content/src/Application/ContentDataMapper/DataMapper/RoutableDataMapper.php

```diff
@@ -165,9 +165,10 @@ public function map(
             );
         }
 
-        if ('/' === $routePath) {
```

Comment:
@alexander-schranz do you think we really need this exception here? 🤔

Reply:
@Prokyonn I don't think so it should `conflict` by unique key if the homepage already exists. So I think we can remove it.